doc/guide/guide-basics.rst

```diff
@@ -383,7 +383,7 @@ Of course, like matrices, multiplying two objects of incompatible shape throws a
     TypeError: Incompatible Qobj shapes
 
 
-In addition, the logic operators is equal `==` and is not equal `!=` are also supported.
+The logic operators "is equal" `==` and "is not equal" `!=` are also supported.
```

Comment:
I would say that "In addition, " adds continuity to the text so I would leave it as it was.